Content.Client/Animations/ReusableAnimations.cs

```diff
@@ -18,7 +18,10 @@ public static void AnimateEntityPickup(IEntity entity, EntityCoordinates initial
             var animatableClone = entity.EntityManager.SpawnEntity("clientsideclone", initialPosition);
             animatableClone.Name = entity.Name;
 
-            var sprite0 = entity.GetComponent<SpriteComponent>();
+            if(!entity.TryGetComponent(out SpriteComponent sprite0))
+            {
+                throw new Exception($"Entity doesn't have a {nameof(SpriteComponent)}!");
```

Comment:
i would rather log here & just return without animating. a function purely for visuals shouldn't kill everything else after it fails since its not vital IMO